### ISSUE TITLE
[Question] How to disable licensecheck for a specific project

### ISSUE THREAD

**Issue by cba64** [porscheinformatik/sonarqube-licensecheck#349]
Hello,

I did not find in the documentation or issues if there is a property that can be set at pom level (or sonar.properties) to disable/ignore/skip license analysis for a specific project.

Is it possible ?

Note that this sentence in the documentation seems incorrect: 

> Activate the rules of this plugin ("License is not allowed", "Dependency has unknown license") in your SonarQube quality profiles - otherwise the plugin is not executed

Indeed, even with those rules disabled in the Quality Profile used, the plugin is executed (according to logs).

Thanks & Regards

**Comment by derkoe** [comment]
The plugin is not executed when it is not active in your quality profiles - make sure to check for all languages.

**Comment by derkoe** [comment]
I think you can also set "-Dlicensecheck.activation=false" to disable the plugin completely.

**Comment by cba64** [comment]
Hello,

Thank you for your replies !

Unfortunately, I tried to double check the Quality Profiles + setting the "-Dlicensecheck.activation=false", but the licenses are stilled scanned although correctly not reported.

I still have a lot of following log lines in the project I want to exclude:

```
[INFO] Sensor License Check [licensecheck]
[INFO] Scanning for Maven dependencies (POM: file:///xxxxxxxxxxxxx/pom.xml)
[INFO] No licenses found in dependency xxxxxxxxx
[INFO] No licenses found in dependency xxxxxxxxx
...
...
[INFO] No licenses found in dependency xxxxxxxxx
[INFO] No licenses found in dependency xxxxxxxxx
[INFO] Sensor License Check [licensecheck] (done) | time=4848ms
```

And it repeats for multiple pom, which leads to a maven build taking almost twice the time to complete.

Would you have any clue about this ?

Thanks & Regards

**Comment by acmraju** [comment]
@derkoe  Facing the same issue. These rules are gettng executed even without enabling them in Quality profile and also even with the option "-Dlicensecheck.activation=false".

**Comment by derkoe** [comment]
@acmraju I have just tried it with our demo project https://github.com/porscheinformatik/angular-spring-heroes. 

When I run
```
mvn -B sonar:sonar -Dlicensecheck.activation=false
```

I see this in the output:
```
[INFO] 19:31:02.423 Sensor License Check [licensecheck]
[INFO] 19:31:02.423 Scanner is set to inactive. No scan possible.
[INFO] 19:31:02.423 Sensor License Check [licensecheck] (done) | time=0ms
...
[INFO] 19:31:07.942 Sensor License Check [licensecheck]
[INFO] 19:31:07.942 Scanner is set to inactive. No scan possible.
[INFO] 19:31:07.942 Sensor License Check [licensecheck] (done) | time=0ms
...
[INFO] 19:31:08.125 Sensor License Check [licensecheck]
[INFO] 19:31:08.125 Scanner is set to inactive. No scan possible.
[INFO] 19:31:08.125 Sensor License Check [licensecheck] (done) | time=0ms
```

So, the plugin is not executed (even though it's active in SonarQube profile).

How do you run it?

**Comment by acmraju** [comment]
@derkoe  

I ran the following Maven command:
```sh
mvn -B sonar:sonar -Dlicensecheck.activation=false
```
I also disabled the license check rule in the quality profiles. Despite these steps, the rule continues to execute.

**Setup Details:**

- We have two independent SonarQube instances.
- Both instances are on the same version of SonarQube and the same version of the plugin.
- The rule behaves as expected on one instance but not on the other.
- On the problematic instance, the rule executes even though it is deactivated in the quality profile and the `-Dlicensecheck.activation=false` option is used.
